projects/Allwinner/bootloader/release

```diff
@@ -9,4 +9,11 @@ mkdir -p "$DSTDIR"
     cp -a "$SRCDIR/u-boot-sunxi-with-spl.bin" "$DSTDIR"
   fi
 
-  cp -a "$SRCDIR"/sun*-${DEVICE,,}-*.dtb "$DSTDIR"
+  case ${DEVICE} in
+    H2+)
+      cp -a "$SRCDIR"/sun*-h[23]-*.dtb "$DSTDIR"
+      ;;
+    *)
+      cp -a "$SRCDIR"/sun*-${DEVICE,,}-*.dtb "$DSTDIR"
+      ;;
+  esac
```

Comment:
I would avoid this change and rather name device folder "H2-plus". I'm not sure "+" in folder name is a good idea anyway.

Reply:
Correct. To avoid downstream issues $DEVICES should (must) be named without special characters.

Reply:
Done